### PR TITLE
CB-11924 - UI and CDPcli status discrepancy

### PR DESCRIPTION
**Pull request by gregito**

### CODE REVIEW

**Comment by keyki** [comment]
I'm not sure if this will work. Salt update is an independent flow and it might not have any other flow attached to it.

File: core/src/main/resources/schema/app/20210413082550_CB-11924_UI_and_CDPcli_status_discrepancy.sql [hunk 1]
```diff
@@ -0,0 +1,25 @@
+-- // CB-11924 UI and CDPcli status discrepancy
+-- Migration SQL that makes the change goes here.
+UPDATE stackstatus
+SET detailedstackstatus = 'SALT_UPDATE_FINISHED'
+AND detailedstackstatus = 'AVAILABLE'
```
Comment (lacikaaa) [inline core/src/main/resources/schema/app/20210413082550_CB-11924_UI_and_CDPcli_status_discrepancy.sql:5]: missing `where`

File: core/src/main/resources/schema/app/20210413082550_CB-11924_UI_and_CDPcli_status_discrepancy.sql [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+SET status = 'UPDATE_IN_PROGRESS'
+WHERE status = 'AVAILABLE'
+AND statusreason = 'Salt update finished.';
+
+
+-- //@UNDO
+-- SQL to undo the change goes here.
+UPDATE stackstatus
+SET detailedstackstatus = 'AVAILABLE'
+AND detailedstackstatus = 'SALT_UPDATE_FINISHED'
```
Comment (lacikaaa) [inline core/src/main/resources/schema/app/20210413082550_CB-11924_UI_and_CDPcli_status_discrepancy.sql:18]: missing `where`

**Review by lacikaaa — CHANGES_REQUESTED** [review]
could you elaborate on what issue and how will this solve?

**Pull request closed without merging**